### ISSUE TITLE
we can't download RHPE datasets or the pretrained models

### ISSUE THREAD

**Issue by dongfangzhizhu** [BCV-Uniandes/Bonet#4]
we can't download RHPE datasets or the pretrained models,I can't open these link,can you help me?

**Comment by mc-escobar11** [comment]
Hello, thank you for the interest in the RHPE Dataset! 

Due to the current global situation, we have been having some technical difficulties with our website services. However, you can download our Bone Age Assessment resources using this link
http://marr.uniandes.edu.co/carpos/images/examples/datasets/ 
[RHPE_dataset.txt](https://github.com/BCV-Uniandes/Bonet/files/5269461/RHPE_dataset.txt)

If you need additional information feel free to email us at mc.escobar11@uniandes.edu.co or ci.gonzalez10@uniandes.edu.co

**Comment by dongfangzhizhu** [comment]
Thank you for replay,but this link is 404 error.
I will email to you 😊

> Hello, thank you for the interest in the RHPE Dataset!
> 
> Due to the current global situation, we have been having some technical difficulties with our website services. However, you can download our Bone Age Assessment resources using this link
> http://marr.uniandes.edu.co/carpos/images/examples/datasets/
> [RHPE_dataset.txt](https://github.com/BCV-Uniandes/Bonet/files/5269461/RHPE_dataset.txt)
> 
> If you need additional information feel free to email us at [mc.escobar11@uniandes.edu.co](mailto:mc.escobar11@uniandes.edu.co) or [ci.gonzalez10@uniandes.edu.co](mailto:ci.gonzalez10@uniandes.edu.co)

**Comment by dakun333** [comment]
> Thank you for replay,but this link is 404 error.
> I will email to you 😊
> 
> > Hello, thank you for the interest in the RHPE Dataset!
> > Due to the current global situation, we have been having some technical difficulties with our website services. However, you can download our Bone Age Assessment resources using this link
> > http://marr.uniandes.edu.co/carpos/images/examples/datasets/
> > [RHPE_dataset.txt](https://github.com/BCV-Uniandes/Bonet/files/5269461/RHPE_dataset.txt)
> > If you need additional information feel free to email us at [mc.escobar11@uniandes.edu.co](mailto:mc.escobar11@uniandes.edu.co) or [ci.gonzalez10@uniandes.edu.co](mailto:ci.gonzalez10@uniandes.edu.co

> we can't download RHPE datasets or the pretrained models,I can't open these link,can you help me?

我能跟你取得联系么？

**Comment by Vivian-double** [comment]
Hello authors. I can't download RHPE datasets. The previous link is no longer accessible. Can you help me?

**Comment by mc-escobar11** [comment]
Hello, 
our webpage has been going through some modifications and is not available at the moment. However, you can access the datasets and annotations through the following links. If you need to evaluate the performance on the test set please send me an email ([mc.escobar11@uniandes.edu.co](mailto:mc.escobar11@uniandes.edu.co)) with a csv file following the same format of the examples ([RHPE](https://drive.google.com/file/d/1tAQW81d3PHmvrS1wPhGmmZZvyS5DrTuK/view?usp=sharing), [RSNA](https://drive.google.com/file/d/1vGF9BtXhzXlAf9Mq3jwEBmqi6JZx6uZf/view?usp=sharing)).

RHPE: [Readme](https://drive.google.com/file/d/1hxAV5trnxTbIxJwoH3UsSxPsSywpWRrF/view?usp=sharing)| [Train](https://drive.google.com/file/d/1Lq3XCOdOkJJKXPVPEYLvf1ScaFGh5uU6/view?usp=sharing)| [Validation](https://drive.google.com/file/d/1bjvC1YZ0h0TK1MXeamWXb8dF6wx-ZKQQ/view?usp=sharing)| [Test](https://drive.google.com/file/d/1sNFFDxbYjIeeBoSeYUokyq4dts_ZgnA_/view?usp=sharing)| [Annotations](https://drive.google.com/file/d/1NOtsunlmnYMufXEeeIHO_DPI77HJzx31/view?usp=sharing)
RSNA:[Train](https://drive.google.com/file/d/1SKmjs8Lvjv8DBs3LSy0XkbWDyJr_U3YF/view?usp=sharing)| [Validation](https://drive.google.com/file/d/17JM6Y5VeVgS35ywTKJ1U5CqHhDsF8irf/view?usp=sharing)| [Test](https://drive.google.com/file/d/1-8akTtn8rAf4cQklIJQ1NUbFnTlGocgc/view?usp=sharing)| [Annotations](https://drive.google.com/file/d/1mOveYqmsiiONFxAUhDCWaFvCnq6GSIRl/view?usp=sharing)
